Source/Core/Core/PowerPC/JitArm64/JitAsm.cpp

```diff
@@ -654,5 +654,29 @@ void JitArm64::GenerateCommonAsm()
   paired_store_quantized[30] = storeSingleS8Slow;
   paired_store_quantized[31] = storeSingleS16Slow;
 
-  GetAsmRoutines()->mfcr = nullptr;
+  // Input in X2, output in W3, clobbers X2-X5 and flags. X0 and X1 are not used, because this is
+  // called from code in JitArm64_Backpatch.cpp which uses those two registers for other purposes.
+  cdts = GetCodePtr();
+
+  UBFX(X4, X2, 52, 11);
+  SUB(W5, W4, 874);
+  CMP(W5, 896 - 874);
```

Comment:
Does this differ from `CMP(W4, 896);`? If not, this has a lower latency.

Reply:
Yes. We want the denormal path if 874 ≤ W4 ≤ 896, not simply if W4 ≤ 896.

Reply:
Indeed, sorry for the noise....